### PR TITLE
fixes #56 and #111, use blocks instead of attachments for list action

### LINKED ISSUE #56: Make the list action format better
Would be nice if the format of the output was a bit more pretty.

We could leverage `attachment` do to that I believe:
https://api.slack.com/docs/message-guidelines

### PR DESCRIPTION
**Pull request by kamigerami**
This fixes #56  and part of #111 .

example output is : 

Screen Shot 2019-10-20 at 13 19 44

### CODE REVIEW

**Comment by parberge** [comment]
Will this really fix #111 ? Don't we still use attachment when sending the interactive message

**Comment by kamigerami** [comment]
> Will this really fix #111 ? Don't we still use attachment when sending the interactive message

It fixes just part of it -> not touching interactive at this point - only list action however the support for "blocks" instead of attachments are there.

File: chalicelib/action.py [hunk 1]
```diff
@@ -161,6 +162,34 @@ def send_response(self, message):
         )
         return ""
 
+    def send_block(self, message):
```
Comment (kamigerami) [inline chalicelib/action.py]: This will send_block instead of attachment which is part of #111 preparation.

**Comment by parberge** [comment]
I'm currently away from home. I would love to do a proper review of this later

**Comment by kamigerami** [comment]
> I'm currently away from home. I would love to do a proper review of this later

I'm sure there are a million naming conventions etc that we could change to make this "better" as well as refactor the code and place some of the functions in different places... at this point .. I think we just want to test that it works before anything else ...

**Comment by parberge** [comment]
> > I'm currently away from home. I would love to do a proper review of this later
> 
> I'm sure there are a million naming conventions etc that we could change to make this "better" as well as refactor the code and place some of the functions in different places... at this point .. I think we just want to test that it works before anything else ...

So I shouldn't review this? This comes down to the problem we have that we can't test slack without merging to master?

**Comment by kamigerami** [comment]
Updated .travis to push to dev on all branches != master

**Pull request merged**